corgidrp/photon_counting.py

```diff
@@ -284,17 +291,18 @@ def get_pc_mean(input_dataset, pc_master_dark=None, T_factor=None, pc_ecount_max
         if pc_master_dark is not None:
             if type(pc_master_dark) is not data.Dark:
                 raise Exception('Input type for pc_master_dark must be a Dataset of corgidrp.data.Dark instances.')
+            if (pc_master_dark.ext_hdr['EXPTIME'], pc_master_dark.ext_hdr['EMGAIN_C']) != unique_vals[0][:2]:
+                raise PhotonCountException('Dark should have the same EXPTIME and EMGAIN_C as input_dataset.')
```

Comment:
it would be useful in the error message to state what exptime/emgain_c we got, vs what was expected.